source/includes/read/count.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+
+#include <bsoncxx/builder/basic/document.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/client.hpp>
+#include <mongocxx/instance.hpp>
+#include <mongocxx/uri.hpp>
+
+using bsoncxx::builder::basic::kvp;
+using bsoncxx::builder::basic::make_document;
+
+int main() {
+    mongocxx::instance instance{};
+    mongocxx::uri uri("<connection string>");
+    mongocxx::client client(uri);
+
+    // start-db-coll
+    auto db = client["sample_training"];
+    auto collection = db["companies"];
+    // end-db-coll
+ 
+    {
+        // Counts documents that have a "founded_year" value of 2010
+        // start-count-accurate
+        auto result = collection.count_documents(make_document(kvp("founded_year", 2010)));
+        std::cout << "Number of documents: " << result << std::endl;
```

Comment:
S: give more descrriptive output
```suggestion
        std::cout << "Number of companies founded in 2010: " << result << std::endl;
```